modules/seqexec/server/src/main/scala/seqexec/server/TranslateSettings.scala

```diff
@@ -9,5 +9,6 @@ final case class TranslateSettings(
   gwsKeywords:   Boolean,
   gcalKeywords:  Boolean,
   gmosKeywords:  Boolean,
-  gnirsKeywords: Boolean
+  gnirsKeywords: Boolean,
+  niriKeywords:  Boolean
```

Comment:
Do we need to update the `app*.conf` files?

Reply:
No, this value is calculated from another app.conf setting, that was already added for Niri.